sound/soc/sof/ipc4-ops.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __SOUND_SOC_SOF_IPC4_OPS_H
+#define __SOUND_SOC_SOF_IPC4_OPS_H
+
+#include "sof-priv.h"
+
+extern const struct ipc_tplg_ops ipc4_tplg_ops;
+/* tplg_ipc4_control_ops is not implemented yet, using tplg_ipc3_control_ops for now */
+extern const struct ipc_tplg_control_ops tplg_ipc3_control_ops;
```

Comment:
I don't really get the comment: it's fine to just declare tplg_ipc4_control_ops but have no callback implemented.

This comment can be really confusing, as if we could use ipc3 in some cases. That will trigger comments for no good reason.